rollup.config.js

```diff
@@ -36,7 +37,7 @@ export default [
 			`src/webpack.ts`
 		],
 		output: {
-			dir: 'dist',
+			dir: path.join(__dirname, 'dist'),
```

Comment:
I usually use `dir : "./dist"` and it works fine in windows, if it helps at all.

Reply:
huh. will try that next. thanks!